### PR TITLE
Fix mPDF, add dependencies

### LINKED ISSUE #587: PDF generation not working correctly in Arabic and Farsi
The webapp requests `https://cms.integreat-app.de/:location/wp-admin/admin-ajax.php` with some parameters (can be seen in https://github.com/Integreat/integreat-webapp/blob/master/src/components/Content/PDFButton.js), but the generated pdf in ar only contains question marks, like in https://web.integreat-app.de/augsburg/ar/%D8%AA%D8%B9%D9%84%D9%85-%D8%A7%D9%84%D8%A3%D9%84%D9%85%D8%A7%D9%86%D9%8A%D8%A9-%D9%81%D9%8A-%D8%A3%D9%88%D8%AC%D8%B3%D8%A8%D8%B1%D8%AC/deutsche-sprache. Probably due to wrong fonts(?).
// Note: There's also a font-parameter where the webapp only sends an empty string to the backend, should we put sth there?

### PR DESCRIPTION
**Pull request by svenseeberg**
* Fixed 2 bugs in ig-mpdf/IntegreatMpdf.php
    * removed / from file_path which lead to an absolute path /wp-content/...
    * updated new mPDF() which does not use it's own namespace anymore
    * Create download directory if it does not exist
* Added dependencies. Sorry for the 47 MB clusterfuck
* Fixes #587 by replacing plugin

### CODE REVIEW

File: wp-content/plugins/ig-mpdf/vendor/mpdf/mpdf/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+vendor/*
```
Comment (timoludwig) [inline wp-content/plugins/ig-mpdf/vendor/mpdf/mpdf/.gitignore]: to stay consistent, we should include this `vendor/`-directory in the repo

Comment (svenseeberg) [reply wp-content/plugins/ig-mpdf/vendor/mpdf/mpdf/.gitignore]: This should be there, see changed files of this commit ...

Comment (timoludwig) [reply wp-content/plugins/ig-mpdf/vendor/mpdf/mpdf/.gitignore]: ok, pdf-generation works now.

**Comment by timoludwig** [comment]
there are still some minor issues with the pdf generation (page title doesn't seem to work in some cases). before approving the new plugin, we probably should do some more testing.

**Comment by svenseeberg** [comment]
I think we should focus our work on making this work. PDF export is one of our core features. But before we merge, we need this to work flawlessly.

**Comment by timoludwig** [comment]
I opened a new PR for this: #684

**Pull request closed without merging**